demos/dws-equity-style-box/demo.js

```diff
@@ -0,0 +1,140 @@
+import { getPostmanFile } from '../utils/postman-localstorage.js';
+
+getPostmanFile(displayEquitySectorsBreakdown, 'postmanEnvironmentDWS');
+
+const loadingLabel = document.getElementById('loading-label');
+
+async function displayEquitySectorsBreakdown (postmanJSON) {
+
+    const connector = new HighchartsConnectors.MorningstarDWS.InvestmentsConnector({
+        postman: {
+            environmentJSON: postmanJSON['postmanEnvironmentDWS']
+        },
+        security: {
+            id: '0P00006W6Q'
+        },
+        converters: {
+            EquityStyleBox: {}
+        }
+    });
+
+    await connector.load();
+
+    const dataTable = connector.getTable('StockStyle'),
+        data = dataTable.getRows(0, 9).map((row) => {
```

Comment:
```suggestion
        data = dataTable.getRows().map((row) => {
```

No need for that if you want to get all of the rows